shared/js/media/video_player.js

```diff
@@ -105,19 +105,25 @@ function VideoPlayer(container) {
   var orientation = 0; // current player orientation
 
   // These are the raw (unrotated) size of the poster image, which
-  // must have the same size as the video.
+  // may have the same size as the video. We may find when we actually
+  // load the video they are different and resize the video element.
   var videowidth, videoheight;
+  var posterwidth, posterheight;
 
   var playbackTime;
   var capturedFrame;
 
-  this.load = function(video, posterimage, width, height, rotate, timestamp) {
+  this.load = function(video, posterimage, width, height, rotate,
+                       posterWidth, posterHeight, timestamp)
```

Comment:
You're changing the signature of a public function here, but I'm pretty sure that media_frame is the only client of this code, so it is probably okay.